### DISCUSSION TITLE
Add support for shared memory

### DISCUSSION THREAD

**Discussion by LDprg** [NVIDIA/open-gpu-kernel-modules#618] [Ideas]
On Windows nvidia drivers have a feature for sharing memory via ram and therefore extending the vram with lower performance ram. In the end this is still better than just running out of vram and alot of tasks/games could really benefit from it.

I would suggest adding support for shared memory under linux too, since it is afaik only currently possible with noveau (on nvidia). It could fix crashes when people run out of ram and increase performance on vram heavy games.

**Comment by LDprg** [comment]
Note that I sometimes need to disable vram heavy effects in games since I am running out of vram. This causes game/system crashes and or really bad performance (sub 30 fps compared to stable 144/60 fps without those effects) and alot of lag spikes.

This wouldn't happen with shared memory (games with those effects work mostly just fine on Windows). So may user could get some better graphics out of their rtx 4060 or similar.

**Comment by bioluks** [comment]
This will get ignored, so I opened an issue since they advertise this as a working feature everywhere. Check https://github.com/NVIDIA/open-gpu-kernel-modules/issues/663

**Comment by linuxiaobai** [comment]
I confirm this is a major issue not resolved.
